build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+# Copyright (c) 2012-2019 Sébastien Rombauts (sebastien.rombauts@gmail.com)
+#
+# Distributed under the MIT License (MIT) (See accompanying file LICENSE.txt
```

Comment:
This header contains info about incorrect license. I think you need to remove it

Reply:
Ok I remove it.